packages/tc-api-rest-handlers/__tests__/absorb.spec.js

```diff
@@ -3,77 +3,62 @@ const { dbUnavailable } = require('../../../test-helpers/error-stubs');
 
 const { absorbHandler } = require('../absorb');
 
+const absorb = absorbHandler();
+
 describe('rest POST (absorb)', () => {
 	const namespace = 'api-rest-handlers-absorb';
 	const mainCode = `${namespace}-main`;
 	const absorbedCode = `${namespace}-absorbed`;
-	const childCode = `${namespace}-child`;
+	const leafCode = `${namespace}-leaf`;
 	const parentCode = `${namespace}-parent`;
 
 	const {
 		createNodes,
 		createNode,
 		connectNodes,
-		meta,
+		stockMetadata,
 		getMetaPayload,
 	} = setupMocks(namespace);
 
-	const documentStore = {};
-
-	beforeEach(() => {
-		documentStore.absorb = jest.fn(async () => ({}));
-	});
-
-	afterEach(() => {
-		jest.resetAllMocks();
-	});
+	const testPayload = {
+		type: 'SimpleGraphBranch',
+		code: mainCode,
+		codeToAbsorb: absorbedCode,
+	};
 
-	const absorb = absorbHandler({ documentStore });
-	const getInput = (override, metadata = {}) =>
```

Comment:
Highlight #6 Removes most (?all) these fancy test input factories in favour of defining some default payloads and using object spread to use these in tests (see 'green' line 23)